### PR TITLE
Development

### PR DESCRIPTION
**Pull request by balen**
Proposed staging build so people can test

99% complete profile screen

### CODE REVIEW

File: app/javascript/components/email_address_editor.vue [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      get: function() {
+        if (this.value.isdefault) {
+          return this.value.isdefault
+        } else {
+          return null
+        }
+      },
+      set: function(val) {
+        // console.debug("****** SET", val)
+        // this.value.isdefault
```
Comment (Gailbear) [inline app/javascript/components/email_address_editor.vue:74]: Is this commented out on purpose?

File: app/javascript/profile/availability_calendar.vue [hunk 2]
```diff
@@ -11,73 +12,117 @@
...
+      )
+      this.dayEvents.push(arg)
+      this.updateAvailabilities()
     },
-    // TODO: this iniates lots of extra events ...
+    updateAvailabilities() {
+      this.update_availability({person: this.currentUser, params: this.dayEvents})
+    },
+    // TODO: this initiates lots of extra events ...
     // can we set scroll top without setting off an event?
```
Comment (Gailbear) [inline app/javascript/profile/availability_calendar.vue:126]: Event.preventDefault()?

File: app/javascript/profile/exclusions_picker.vue [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+          {{ exclusion.title }}
+        </b-form-checkbox>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import settingsMixin from "@/store/settings.mixin";
+import personExclusionMixin from '../store/person_exclusion.mixin'
```
Comment (Gailbear) [inline app/javascript/profile/exclusions_picker.vue:20]: In general we should prefer @/ to ../

**Comment by Gailbear** [comment]
I skimmed. Looks good generally!

**Review by Gailbear — APPROVED** [review]

File: app/javascript/components/email_address_editor.vue (hunk 1 above)
Comment (balen) [reply app/javascript/components/email_address_editor.vue:74]: Yes. Vue complains when there is no set function in a computed property. But we are not setting it (all is done via events etc)

File: app/javascript/profile/availability_calendar.vue (hunk 2 above)
Comment (balen) [reply app/javascript/profile/availability_calendar.vue:126]: prevent default will cause the other scroll bars to not be in sync (I tried that). We need them to communicate with each other. It is not broken, just annoying from a coder PoV

**Pull request merged**